CHANGELOG

```diff
@@ -1,6 +1,18 @@
 devel
 -----
 
+* APM-294: Added telemetrics API that gathers info about a deployment for 
+  the receiving side to generate statistics. It's accessible via 
+  `_admin/telemetrics`. It's on by default, only disabled in maintainer 
+  mode,  and can be turned on/off with server startup parameter 
+  `--server.send-telemetrics` set to false. 
+  If set to true, every time the user logs into arangosh, one request is
+  sent to gather telemetrics from `_admin/server-info`. Only users that are 
+  admin might get a successful http response with the telemetrics object.
+  To send telemetrics to our endpoint for testing in maintainer mode, 
+  the shell must be started with parameter 
+  `--telemetrics.send-to-endpoint` set to true.
```

Comment:
```suggestion
* APM-294: Added telemetrics API that gathers anonymous feature usage
  statistics from a deployment. The API is accessible via the endpoint
  `/_admin/telemetrics`. The API is enabled by default in release builds, but
  disabled by default in maintainer mode. It can be explicitly turned on/off 
  with the server startup parameter `--server.send-telemetrics`. 
  The required access privileges to access the telemetrics API can be 
  configured via the server startup option `--server.support-info-api`.
  The telemetrics API is used by the arangosh: every time the arangosh is 
  started, it will send a request to the connected server to gather telemetrics
  from the `/_admin/server-info` endpoint. The telemetrics data are then 
  sent to an aggregation service that is run by ArangoDB.
```